cvat-ui/src/components/common/not-found.tsx

```diff
@@ -2,26 +2,51 @@
 //
 // SPDX-License-Identifier: MIT
 
+
 import React from 'react';
 import Result from 'antd/lib/result';
+import Button from 'antd/lib/button';
+import { useHistory } from 'react-router';
 
-export const JobNotFoundComponent = React.memo((): JSX.Element => (
-    <Result
-        className='cvat-not-found'
-        status='404'
-        title='Sorry, but this job was not found'
-        subTitle='Please, be sure information you tried to get exist and you have access'
-    />
-));
+export const JobNotFoundComponent = React.memo((): JSX.Element => {
+    const history = useHistory();
+    const handleGoToJobs = () => {
+        history.push('/jobs');
+    };
+    return (
+        <Result
+            className='cvat-not-found'
+            status='404'
+            title='Sorry, but this job was not found'
+            subTitle='Please, be sure information you tried to get exist and you have access'
+            extra={
+                <Button type='primary' onClick={handleGoToJobs}>
+                    Go to Jobs (reset filters)
+                </Button>
+            }
+        />
+    );
+});
```

Comment:
This should be covered in #10028 with other not found components, I suggest to remove it